tardis/interfaces/siteadapter.py

```diff
@@ -205,6 +226,29 @@ async def resource_status(
         """
         raise NotImplementedError
 
+    @property
+    @lru_cache(maxsize=None)
```

Comment:
Is an unbounded cache actually needed here? I assume a small cache (~16-32) should catch reasonably many lookups without being unlimited, but I'm not sure how this is used.

Reply:
You are right. Maxsize can be reduced.  I will set it to 16 then.